content/conduct.md

```diff
@@ -3,8 +3,24 @@ title: Code of Conduct
 type: page
 ---
 
-# Code of conduct
+# Code of Conduct
+
+We've found that code reading clubs work best when you involve people of different backgrounds and levels of experience. People read code differently and we can learn a lot from each other! It's important for us to make sure the environment where these discussions happen is inclusive and supportive. Everyone should feel comfortable to participate! The following guidelines are meant to codify these intentions.
 
 ## How to be with each other
+
+> a coherent paragraph about "no harassment" that I was unable to write
+
+> do we want to list the Recourse Center guidelines? (“no feigning surprise”,“no well-actuallys”, “no backseat driving”, and “no subtle -isms”). I like it and it mostly applies (the backseat driving one is irrelevant because everyone mostly has the same context)
+
+> I'd also like to add a point to avoid critisizing the person who wrote the code, and to hold place for unknown context or reasons for weird code structure etc., especially if the code was written by one of the attendees.
+
 ## Guidelines for running a club
+
+The code of conduct above is the one we use in our clubs. Feel free to use it as is or modify it to fit your circumstances.
+
+As a facilitator it's important that you not only follow your code of conduct, but also enforce it. Your code of conduct should be made available to attendees before the club and you should ask them to read it before attending. It's good practice to quickly go over the guidelines at the beginning of the session. Make sure to give attendees a way to inform you of any code of conduct violations they experience.
+
 ## Talk to us
+
+> is this section for people in our clubs who need us to deal with a CoC issue, or for people who want to run a club and have questions?
```

Comment:
I think I intended it as a section for people who want us to deal with a CoC issue or have a related concern / suggestion.